ROADMAP.md

```diff
@@ -0,0 +1,198 @@
+
+## 3. The knowledge required to work efficiently with Testground is available and easy to access
+
+**Why:** We believe test is valuable when everyone in the team can contribute to testing. The platform has to be approachable.
+
+Related - [EPICS 1741](https://github.com/testground/testground/issues/1471).
+
+
+### Milestone 1: There is a working quickstart that is the entry point to every other documentation
+
+
+### Milestone 2: Testground has an official release process
+
+- We distribute versions following an explicit release process (no more `:edge` by default)
+- It's easy to follow changes (CHANGELOG)
+- We distribute binaries
+
+
+### Milestone 3: Testground provides extensive documentation
+
+- The documentation is up-to-date
+    - Generates configuration documentation
+- We provide introduction guides for every language
+- We provide doc for Common Patterns
+
+
+### Milestone 4: Potential users know about Testground's existence and features
+
+- "Public Relations".
+
+
+## 4. Testground Developement follows high-standard
+
+**Why:** Testground has already provided value amongst many different projects. As it grows it will expand in different directions and it's critical that we can support these with efficient developement processes and project management. Being an open source project, external contribution is critical.
+
+### Testground CI & Testing is practical and reliable
+
+(I feel like this should be a top-level point)
+
+- We use a single CI (GitHub CI)
+- Refactor the testing to be easier to use and simpler to maintain
+    - remove shell scripts
+- Plan for testing EKS
+- Measure & remove flakiness
+
+
+### I have all the documentation I need as a Maintainer
+
+- public specifications
+- public discussions
+- community gatherings
+
+
+### Efficient Project Management and Project Visibility
+
+- We have a single maintainer team for review,
+- We have a clear label and triaging process,
+- We have a clear contribution process (protected branches, etc),
+- We have precise project management tooling (re-ignite ZenHub?)
+
+
+## 5. Testground provides the networking tooling required to test complex Distributed / Decentralized Applications
+
+**Why:** This is a main selling point for Testground.
+
+### Milestone 1: ExplicitNetwork Simulation Support
+
+There is a clear matrix of what features can be used with which Testground runner.
+
+- MTU issues, networking simulation, etc.
+- This will be a Matrix of feature x runner x precision (transport level, application level, etc.).
+
+
+### Milestone 2: Testground provides support for important networks topologies
+- Access to public networks - [issue 1472](https://github.com/testground/testground/issues/1472)
+- NAT simulation - [issue 1299](https://github.com/testground/testground/issues/1299)
+- Complex topologies - [issue 1354](https://github.com/testground/testground/issues/1354)
+
+
+### Milestone 3: Testground provides a way to run tests over real networks
```

Comment:
Does this depend on the other milestones, or is this a prioritization? Should this be its own top-level goal?

Reply:
(resolving, roadmap was rewritten with clearer ordering)